gui/default/syncthing/core/syncthingController.js

```diff
@@ -1586,11 +1607,17 @@ angular.module('syncthing.core')
 
         $scope.saveIgnores = function () {
             if (!$scope.editingExisting) {
+                $scope.newIgnores.edited = true;
+                $scope.newIgnores.ignores = $('#editIgnores textarea').val().split('\n')
                 return;
```

Comment:
I'd just have a different button with a different callback if not editing existing